optaplanner-core/src/test/java/org/optaplanner/core/config/heuristic/selector/entity/EntitySelectorConfigTest.java

```diff
@@ -39,9 +37,11 @@ public void phaseOriginal() {
         EntitySelector entitySelector = entitySelectorConfig.buildEntitySelector(
                 buildHeuristicConfigPolicy(),
                 SelectionCacheType.JUST_IN_TIME, SelectionOrder.RANDOM);
-        assertInstanceOf(FromSolutionEntitySelector.class, entitySelector);
-        assertNotInstanceOf(ShufflingEntitySelector.class, entitySelector);
-        assertEquals(SelectionCacheType.PHASE, entitySelector.getCacheType());
+        assertThat((Object) entitySelector)
```

Comment:
Redundant cast. It seems this is done automatically by your IDE. Please check if you can avoid that.